apps/mull-ui/src/app/components/event-page-info/event-page-info.scss

```diff
@@ -0,0 +1,28 @@
+@import 'variables';
+
+.event-page-info-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.info-row {
+  display: flex;
+  padding: 0.5rem 1rem;
+  border-bottom: 0.5px solid rgba($color: $primary-color, $alpha: 0.5);
+}
+
+.event-page-button {
+  align-self: center;
+  margin: 0.5rem 0;
```

Comment:
Button could use more margin-top to match the mockup and look better.

Reply:
Yup agree, top and bottom should have more margin